src/content/cloud/container-resources.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: Container Resources
+description: Optimizing Container Resource Allocation for Efficient Development Environments
+sidebar_label: Container Resources
+id: container-resources
+---
+
+import Image from '@theme/Image';
+
+Okteto leverages the concept of Kubernetes resources to decide on which node to place each container.
+When you specify a resource `limit` for a container, Okteto enforces those limits so that the running container is not allowed to use more of that resource than the limit you set.
+Okteto also reserves at least the `request` amount of that system resources specifically for that container to use.
```

Comment:
I thought we were not doing this, and instead applying a ratio to the limit no? e.g  In cloud we don't let people request 4CPUs right?

Reply:
We only apply the ratio limit in Cloud, not in other environments. I will mention that